source/helpers/components/anchors.ts

```diff
@@ -0,0 +1,38 @@
+import Token from 'markdown-it/lib/token';
+import type anchor from 'markdown-it-anchor';
+
+// Custom permalink function, inspired by and modification of linkInsideHeader
+// More context on custom permalink fns: https://github.com/valeriangalliat/markdown-it-anchor#custom-permalink
+// linkInsideHeader function: https://github.com/valeriangalliat/markdown-it-anchor/blob/649582d58185b00cfb2ceee9b6b4cd6aafc645b7/permalink.js#L76
+export const renderPermalink: anchor.PermalinkGenerator = (
+  slug,
+  opts: anchor.LinkInsideHeaderPermalinkOptions,
+  state,
+  idx,
+) => {
+  opts.ariaHidden = false;
+
+  const title = state?.tokens[idx + 1].children
+    ?.filter(
+      (token: Token) => token.type === 'text' || token.type === 'code_inline',
+    )
+    .reduce((acc, t) => acc + t.content, '');
+
+  const linkTokens = [
+    Object.assign(new state.Token('link_open', 'a', 1), {
+      attrs: [
+        ['class', 'anchor'],
+        ['href', `#${slug}`],
+        ...(opts.ariaHidden ? [['aria-hidden', 'true']] : []),
+      ],
+    }),
+    Object.assign(new state.Token('html_inline', '', 0), {
+      content: `<span class="visuallyhidden">${
+        title ? title : 'section'
+      } permalink'</span>`,
```

Comment:
this "’" mark is showing up when I read the voiceover prompts. For instance, 
![Screenshot 2023-04-25 at 11 30 24 PM](https://user-images.githubusercontent.com/1581694/234470428-0eb32363-9443-42ea-a0b2-8ca716c92dfe.jpg)